test/parallel/test-fs-read-stream-fd.js

```diff
@@ -1,23 +1,23 @@
 'use strict';
 const common = require('../common');
-var fs = require('fs');
-var assert = require('assert');
-var path = require('path');
-var file = path.join(common.tmpDir, '/read_stream_fd_test.txt');
-var input = 'hello world';
-var output = '';
-var fd, stream;
+const fs = require('fs');
+const assert = require('assert');
+const path = require('path');
+const file = path.join(common.tmpDir, '/read_stream_fd_test.txt');
+const input = 'hello world';
 
+let output = '';
 common.refreshTmpDir();
 fs.writeFileSync(file, input);
-fd = fs.openSync(file, 'r');
 
-stream = fs.createReadStream(null, { fd: fd, encoding: 'utf8' });
-stream.on('data', function(data) {
+const fd = fs.openSync(file, 'r');
+const stream = fs.createReadStream(null, { fd: fd, encoding: 'utf8' });
+
+stream.on('data', common.mustCall((data) => {
   output += data;
-});
+}));
 
-process.on('exit', function() {
+process.on('exit', () => {
   fs.unlinkSync(file);
```

Comment:
While we're in here editing the file, can you please remove `fs.unlinkSync(file);`?  (There's no need for tests to clean up the temp dir on exit as all tests that use the temp dir should refresh it before using it.)